### ISSUE TITLE
Submodule 'packaging' isn't included in version 2.4.0

### ISSUE THREAD

**Issue by 3ter** [zjosua/anki-mc#44]
Restarting Anki after updating to 2.4.0 leads to:

> When loading '⁨Multiple Choice for Anki⁩':
⁨Traceback (most recent call last):
  File "aqt/addons.py", line 210, in loadAddons
  File "/home/3ter/.local/share/Anki2/addons21/1566095810/__init__.py", line 37, in <module>
    from .config import *
  File "/home/3ter/.local/share/Anki2/addons21/1566095810/config.py", line 43, in <module>
    from .packaging.packaging import version
ModuleNotFoundError: No module named '1566095810.packaging'

```
~/.local/share/Anki2/addons21/1566095810
$ tree
.
├── config.py
├── __init__.py
├── LICENSE.txt
├── manifest.json
├── meta.json
└── template.py
```

**Comment by TrunkneFlut** [comment]
Have the same problem, shortly after everything went fine for a while without problems

**Comment by 3ter** [comment]
@TrunkneFlut Although the addon dies with an exception it shouldn't impede the multiple choice cards in working correctly if they did so before.

It should only effect that the note type won't be updated (and of course this beautiful message on startup :wink: ).

**Comment by TrunkneFlut** [comment]
You’re absolutely right. It’s just that it still doesn’t work fully (e.g.,
it doesn’t show answers after editing a card) which I misinterpreted for a
second as being related to the error message. Sorry!

**Comment by markus089** [comment]
exact same problem here

**Comment by markus089** [comment]
thank you for your reference but as im not that familiar with github and using its command line i just wait for an updated version. thx anyways.

**Comment by zjosua** [comment]
Should be fixed in v2.4.1.
